pkg/kubelet/network/dns/dns_test.go

```diff
@@ -42,7 +41,9 @@ import (
 )
 
 var (
-	fetchEvent = func(recorder *record.FakeRecorder) string {
+	testHostNameserver = "8.8.8.8"
```

Comment:
Why are these being elevated?  Only used in one place.

Reply:
It's also being used in [pkg/kubelet/network/dns/dns_windows_test.go](https://github.com/kubernetes/kubernetes/pull/110566/files#diff-355f068daa5e904df53af3047861a89f53ad4530b79bb2c121533000ac49b552) changes introduced in this PR. It's easier to keep things consistent.

Reply:
comments?

Reply:
Done